helm/hpcc/templates/_helpers.tpl

```diff
@@ -79,7 +79,7 @@ storage:
   planes:
 {{- /*Generate entries for each data plane (removing the pvc).  Exclude the planes used for dlls and dali.*/ -}}
 {{- range $plane := $planes -}}
- {{- if or (not $plane.labels) (has "data" $plane.labels) }}
+ {{- if or (not $plane.labels) (or (has "data" $plane.labels) (has "lz" $plane.labels)) }}
```

Comment:
This means all "lz" planes are mounted by all engines.
Should "lz" planes only be accessible to dfuserver?

That doesn't mean to say they can't be both, labels can be : ["data", "lz"]

Incidental thought, I wonder if mounting many planes etc., causes latency when spinning up pods for jobs.

Reply:
We are already mounting unnecessary planes e.g dllplanes in dfuserver.  Should generateGlobalConfigMap have types of planes and labels that should be mounted?

Reply:
Mounting unneccessary planes should be avoided, I think. Referencing them in the global config file doen;t matter, but mounting them does.

Reply:
actually, my 1st statement was wrong, this won't mean additional mounts, this is just config.
The data mounts themselves of course are generated via addDataVolumeMount/addDataVolume, and only mount "data" planes.

I don't think it's a big issue, it just means that "lz" planes are seen in the planes list by everyone, not just those that need it (which may just be dfuserver's).

>We are already mounting unnecessary planes e.g dllplanes in dfuserver.

Separate issue, but where? I don't see addDllVolume* used in dfuserver.yaml